packages/rocketchat-ui/client/lib/msgTyping.js

```diff
@@ -43,7 +50,7 @@ export const MsgTyping = (function() {
 			timeouts[room] = null;
 		}
 		const user = Meteor.user();
-		return RocketChat.Notifications.notifyRoom(room, 'typing', user && user.username, false);
+		return RocketChat.Notifications.notifyRoom(room, 'typing', shownName(user), false);
```

Comment:
@vynmera this broke the notifications in some cases.
Sometimes, `user` is missing completely - I believe if nobody is typing.
The earlier `user && ...` would evaluate to false, not notifying the stream - this was intended. Please restore this before merging